independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/AnnotationLiteralProcessor.java

```diff
@@ -381,7 +383,7 @@ public boolean equals(Object o) {
 
         @Override
         public int hashCode() {
-            return Objects.hash(annotationClass.name());
+            return Objects.hashCode(annotationClass.name());
```

Comment:
`Objects.hash(...)` uses varargs so a new array is instantiated each time we call it, which is completely useless in this case.